README.md

```diff
@@ -1,12 +1,41 @@
+This repo contains a set of resources, tutorials and general best practices that our team has developed and continues to refine for effectively collaborating as a group of data scientists. Our team relies heavily on the [Open Data Hub](https://opendatahub.io/) (ODH) project (which we also recommend), so many of our examples will use that toolbox. But our hope is that there is enough generally applicable content that this information can be helpful to those outside the ODH ecosystem. Further, we invite others to [contribute](https://github.com/aicoe-aiops/data-science-workflows/issues/new) their best practices and implementation alternatives : )
 
-* **[Get Started](docs/project_structure/getting-started.md)**
-* **[How to Contribute](docs/project_structure/how-to-contribute.md)**
-* **[Data Science Projects](https://www.operate-first.cloud/data-science)**
-* **[Jupyter Book](https://aicoe-aiops.github.io/data-science-workflows/)**
+## Setup Your Environment
 
-### Contact
+* [What is the Open Data Hub and Operate First?](/docs/setup_environment/odh_and_opf.md) 
+* [Access JupyterHub](/docs/setup_environment/JH_access.md)
+* [Manage data with remote storage](/docs/setup_environment/remote-storage.md)
+* [Setup CI](/docs/setup_environment/thoth.md)
 
-This project is maintained as part of the [Operate First](https://www.operate-first.cloud/) and AIOps teams in Red Hat’s AI CoE as part of the Office of the CTO. More information can be found at https://www.operate-first.cloud/.
+## Develop + Collaborate 
+* [Use Thoth tooling to enhance development](/docs/develop_collaborate/thoth-tools.md)
+* [Best practice for contributing as a Data Scientist](/docs/develop_collaborate/how-to-contribute.md)
+* [Build ML pipelines from notebooks](/docs/develop_collaborate/automated-pipelines.md)
+* [Track your metrics and experiments](/docs/develop_collaborate/track-metrics-using-kubeflow.md)
+* [Share reproducible notebook images](/docs/develop_collaborate/create_and_deploy_jh_image.md)
+* [Quickly deploy interactive environments and JupyterBooks](/docs/develop_collaborate/meteorize.md)
+* [Create interactive dashboards to visualize results](/docs/visualization/trino_and_superset.md)
+
+## Serving + Monitoring
+* [Monitor JupyterHub environment workloads](/docs/serving_monitoring/monitor-jh)
+* [Serve your model with Seldon](/docs/serving_monitoring/deploy-models-using-seldon.md)
+* [Create custom serving images with Seldon](/docs/serving_monitoring/deploy-custom-model.md)
+
+## Project Organization + Structure
+* [Tips for starting a new ML project from scratch](/docs/project_structure/getting-started.md)
+* [Recommendations for structuring an E2E ML project](/docs/project_structure/project_structure.md)
```

Comment:
```suggestion
* [Recommendations for structuring an E2E ML project](/docs/project_structure/project-structure.md)
```
@MichaelClifford, Just pointing out a small typo. Otherwise lgtm. :)

Reply:
thanks! fixed